tests/Integration/Cache/RedisCacheLockTest.php

```diff
@@ -72,6 +72,24 @@ public function test_concurrent_redis_locks_are_released_safely()
         $this->assertFalse(Cache::store('redis')->lock('foo')->get());
     }
 
+    public function test_redis_locks_with_failed_block_callback_are_released()
+    {
+        Cache::store('redis')->lock('foo')->forceRelease();
+
+        $firstLock = Cache::store('redis')->lock('foo', 10);
+        try {
+            $firstLock->block(1, function () {
+                throw new \Exception('failed');
+            });
+        } catch (\Exception $e) {
+            // Not testing the exception, just testing the lock
+            // is released regardless of the how the exception
+            // thrown by the callback was handled.
+        }
+        $secondLock = Cache::store('redis')->lock('foo', 1);
```

Comment:
Add a white line after the catch block.

```suggestion

        $secondLock = Cache::store('redis')->lock('foo', 1);
```